spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -387,6 +387,28 @@ will run in the context of an LRA and controls whether or not:
 - to throw an exception if the method returns particular HTTP status codes.
 - to enlist the class as a participant of the LRA.
 
+These operations are performed before the execution proceeds to the invoked
```

Comment:
yes, should that be somehow interlinked?

Reply:
Well some of the "operations" are performed after the method finishes execution which implies that your statement is unrelated to the items listed above since it does not apply to the 3rd and 5th bullets.